source/pkcore/abilitypages/chi/quickstrike.rst

```diff
@@ -1,32 +1,62 @@
-.. _quickstrike:
-
-===========
+.. quickstrike:
+###########
 QuickStrike
-===========
+###########
+
+Description
+###########
+
+QuickStrike is a Chi Ability.
 
-QuickStrike is a main offensive ability for chiblocking, the user precisely strikes an entity as to improve
-their chances of blocking the entity's chi so they can't bend.
+QuickStrike enables a chiblocker to quickly strike an enemy, potentially blocking their chi. GIF TO BE ADDED
```

Comment:
Change to `QuickStrike allows Chiblockers to quickly damage an enemy's pressure points, dealing increased damage and resulting in a greater chance to block their chi. GIF TO BE ADDED`